CMakeLists.txt

```diff
@@ -109,13 +291,17 @@ add_libnanomsg_test (zerocopy 5)
 add_libnanomsg_test (shutdown 5)
 add_libnanomsg_test (cmsg 5)
 add_libnanomsg_test (bug328 5)
-add_libnanomsg_test (win_sec_attr 5)
+
+# Platform-specific tests
+if (WIN32)
+    add_libnanomsg_test (win_sec_attr 5)
+endif()
```

Comment:
Btw, we should carefully go through the tests to make sure that the autoconf test list is fully represented here.  I think I recall seeing some tests were missing from the CMake build.  Probably through oversight.